ui/lib/pki/addon/templates/error.hbs

```diff
@@ -0,0 +1,48 @@
+<PageHeader as |p|>
+  <p.top>
+    <Page::Breadcrumbs @breadcrumbs={{this.breadcrumbs}} />
+  </p.top>
+  <p.levelLeft>
+    <h1 class="title is-3">
+      <Icon @name="file-text" @size="24" class="has-text-grey-light" />
+      {{this.title}}
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+<div class="tabs-container box is-bottomless is-marginless is-fullwidth is-paddingless">
+  <nav class="tabs" aria-label="secret tabs">
+    <ul>
+      {{#each this.tabs as |oTab|}}
+        <LinkTo @route={{oTab.route}} data-test-tab={{oTab.route}}>
+          {{oTab.label}}
+        </LinkTo>
+      {{/each}}
+    </ul>
+  </nav>
+</div>
+
+<div class="box is-sideless has-background-white-bis has-text-grey has-text-centered has-tall-padding" data-test-pki-error>
+  {{#if (eq this.model.httpStatus 404)}}
+    <h1 class="title is-3 has-text-grey">
+      404 Not Found
+    </h1>
+    <p>Sorry, we were unable to find any content at <code>{{or this.model.path this.path}}</code>.</p>
+  {{else if (eq this.model.httpStatus 403)}}
+    <h1 class="title is-3 has-text-grey">
+      Not authorized
+    </h1>
+    <p>You are not authorized to access content at <code>{{or this.model.path this.path}}</code>.</p>
+  {{else}}
+    <h1 class="title is-3 has-text-grey">
+      Error
+    </h1>
+    <p>
+      {{#if this.model.message}}
+        <p>{{this.model.message}}</p>
+      {{/if}}
+      {{#each this.model.errors as |error|}}
+        <p>{{error}}</p>
+      {{/each}}
+    </p>
+  {{/if}}
+</div>
```

Comment:
This is something I hadn't considered within an engine. I'm wondering if this could be moved to a component in the core addon for wider use and consistency across engines that might need it?

Reply:
This was sort of a rogue change on my part, and I didn't put it in the addon as a component because we already have a components in Vault that *should* cover this use case, except they didn't move gracefully into addons for use in engines. 

I'm not opposed to making a generic-ish component, but I think it should be as a part of cleaning up the existing ones: `BlockError`, `NotFound` (why does it take an entire model 😭), `Clients::Error`, `Error` template can/should all use or be replaced by the new generic addon error component